.github/workflows/flow-linux-platforms.yml

```diff
@@ -59,34 +64,83 @@ on:
         description: 'Whether to run standalone tests'
         type: boolean
         default: true
+      redis-ref:
+        type: string
 
 jobs:
   get-required-envs:
     runs-on: ubuntu-latest
     outputs:
-      platform: ${{ steps.get_platforms.outputs.platform }}
+      platforms_arm: ${{ steps.get_platforms.outputs.arm-platforms }}
+      platforms_x86: ${{ steps.get_platforms.outputs.x86-platforms }}
+      include_arm: ${{ steps.get_platforms.outputs.arm-include }}
+      include_x86: ${{ steps.get_platforms.outputs.x86-include }}
     steps:
       - name: Get platform
         id: get_platforms
+        shell: python
         run: |
-          if [ "${{ inputs.platform }}" == "all" ]; then
-            echo "platform=${{ env.ALL_IMAGES }}" >> $GITHUB_OUTPUT
-          else
-            echo "platform=['${{ inputs.platform }}']" >> $GITHUB_OUTPUT
-          fi
+          import os
```

Comment:
Not sure about the outputs but it might work.
But I like to think of this file as the designated file for this script. I can't see it being used in other places, and I'll also be forced to checkout and fetch the file while currently it is built-in and doesn't require to do so (and deal with additional steps and dependencies)